docs/reference/readers.md

```diff
@@ -103,7 +103,11 @@ Be careful when enabling multiple related formats together.
 
 ### EAN-2 and EAN-5 Supplements
 
-The default `ean_reader` does not read extensions like [EAN-2](https://en.wikipedia.org/wiki/EAN_2) or [EAN-5](https://en.wikipedia.org/wiki/EAN_5) (additional digits printed beside the main barcode).
+**EAN/UPC are the only barcode formats that support supplements.** The EAN-2 and EAN-5 supplements are additional digits printed beside the main barcode, typically used for:
```

Comment:
s/\*\*EAN\/UPC are the only barcode formats that support supplements.\*\*/The EAN and UPC barcode formats support a supplement format, adding an additional 2 or 5 digits beyond the main barcode, EAN-2 and EAN-5, respectively.  They are typically used for:

... something more like that.